src/User/AvatarValidator.php

```diff
@@ -82,4 +92,9 @@ protected function getAllowedTypes()
     {
         return ['jpg', 'png', 'bmp', 'gif'];
     }
+
+    protected function getRules()
+    {
+        return ['avatar' => 'required|file'];
+    }
```

Comment:
Don't we manually check for these ?

Actually we only manually check that it's required, I don't think we check that it's an actual file 🤔 so it would make sense, is this correct?

Reply:
Oh this is code from some testing j was doing... This change shouldnt be here.

Reply:
Alright then, I've logged what I was referring to separately so as not to go out of scope for this PR.